### PR TITLE
Update Profile Page with Hack 2022 Designs

### LINKED ISSUE #404: Update Profile Page
1. Update background to cloud gradient from Figma.
2. Remove Discord username, team matching status, time zone, and skills portion. 
3. Remove QR Code
    1. We can still keep the `UIImage` and `CIImage` code in the `Extensions` folder for generating a QR code for future purposes.
4. Add Default Profile Picture
    1. Since the profile endpoint currently returns the profile pictures from last year, we will have to figure out a way to still show this year's profile pictures (see Figma). One idea is creating a mapping between last year and this year's pictures in `Constants.swift`.
    2. Don't forget to add the new profile pictures in `Assets`!
5. Update the points container.
    1. For the Current Tier text, just display "Current Tier" until the API is updated.
6. Change whatever colors need to be changed.
7. Remove button for editing the profile.

### PR DESCRIPTION
**Pull request by ashayp22**
## Description

_Please ignore the branch name. I do not know what was going on in my head when I created this branch_

- Removes QR code.
- Addes Hack 2022 Designs to HIProfileViewController.
- Add left aligned title to HIBaseViewController.
- Removed the ability to edit User Profile.

Fixes #404 

## Todos

- Need to update the "Current Tier" text with the user's actual tier. API still needs to add this to the Profile endpoint.
- Update the logout button and text if a guest is signed in.

## Screenshots

iPad Pro 11 inch

Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2022-01-19 at 22 20 59

iPhone 12 Pro

Simulator Screen Shot - iPhone 12 Pro - 2022-01-19 at 22 18 46

iPhone SE

Simulator Screen Shot - iPhone SE (2nd generation) - 2022-01-19 at 22 17 24

### CODE REVIEW

File: HackIllinois/ViewControllers/HIProfileViewController.swift [hunk 1]
```diff
@@ -20,78 +20,36 @@ class HIProfileViewController: HIBaseViewController {
...
+    private let profilePointsLabel = HILabel(style: .profileNumberFigure) {
         $0.text = ""
     }
-    private let profileDiscordImageView = HIImageView()
-    private let profileDiscordUsernameView = HILabel(style: .profileUsername) {
-        $0.text = "Discord Username"
+    private let profileTierLabel = HILabel(style: .profileTier) {
+        $0.text = "Current Tier"
     }
     private let profileUsernameView = UIStackView() // Will be initialized in ViewController extension (.axis = .horizontal)
```
Comment (jeffkim2001) [inline HackIllinois/ViewControllers/HIProfileViewController.swift:52]: We do not need profileUsernameView anymore.

File: HackIllinois/ViewControllers/HIProfileViewController.swift [hunk 2]
```diff
@@ -125,228 +94,71 @@ extension HIProfileViewController {
...
-        profileDescriptionView.text = profile.info
-        profileDiscordUsernameView.text = profile.discord
-        interests = profile.interests
-        profileInterestsView.reloadData()
-        profileInterestsViewHeight.constant = profileInterestsView.collectionViewLayout.collectionViewContentSize.height + 20
-
-        profileDiscordImageView.image = #imageLiteral(resourceName: "DiscordLogo")
-
-    }
+        profilePointsLabel.text = "\(profile.points)"
```
Comment (jeffkim2001) [inline HackIllinois/ViewControllers/HIProfileViewController.swift:156]: Append the string "Points" after the value in the profilePointsLabel (example: "480 Points").

**Review by jeffkim2001 — COMMENTED** [review]
Besides some small line comments,
I am unable to view the full profile in the iPhone SE.

Screenshot: 
profile-se

Ensure that the full profile is visible on the smallest possible device, or bring back the scroll view that makes the profile scrollable.

**Comment by ashayp22** [comment]
> Besides some small line comments, I am unable to view the full profile in the iPhone SE.
> 
> Screenshot: profile-se
> 
> Ensure that the full profile is visible on the smallest possible device, or bring back the scroll view that makes the profile scrollable.

@jeffkim2001 interesting, the full profile is viewable on my iPhone SE simulator. Maybe because it is 2nd generation?

**Review by jeffkim2001 — APPROVED** [review]
Looks good to me!

**Pull request merged**